package.json

```diff
@@ -125,6 +125,7 @@
     ]
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
@FreemRL Why do you need this package?

Reply:
I can remove it if you want, I added it to fix this warning:

![image](https://github.com/reactplay/react-play/assets/66525499/2035f2bd-36d0-4067-9a54-03c917f2a023)